.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
do you consider weekly?

Reply:
I think daily makes things more continuous, rather than taking a batch of things regularly?

Reply:
My reasoning is weekly is frequent enough and may save us from getting a bad release.

Reply:
I don't mind that much. Up to you :)